src/apps/experimental/components/LibraryIcon.tsx

```diff
@@ -11,7 +11,7 @@ import Collections from '@mui/icons-material/Collections';
 import Queue from '@mui/icons-material/Queue';
 import Folder from '@mui/icons-material/Folder';
 import React, { FC } from 'react';
-import { CollectionType } from 'types/collectionType';
+import { CollectionType } from '@jellyfin/sdk/lib/generated-client/';
```

Comment:
Imports from the SDK should use the full file path. We have had issues in the past with tree-shaking not working and the entire SDK being included in the final JS bundle.

```suggestion
import { CollectionType } from '@jellyfin/sdk/lib/generated-client/models/collection-type';
```

Reply:
I only commented on this one, but the rest should be updated also. 😅

Reply:
Yeah sorry, needed PC to update all, I've hopefully done it now and not missed any :P